pkg/forwarder/forwarder_test.go

```diff
@@ -2,6 +2,7 @@
 // under the Apache License Version 2.0.
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2017 Datadog, Inc.
+// +build !windows
```

Comment:
why we don't test the forwarder on windows?

Reply:
I agree, this should get tested as well on windows, no?